packages/dev-server/package.json

```diff
@@ -41,9 +41,7 @@
     "test:watch": "mocha \"test/**/*.test.mjs\" --watch --watch-files src,test --reporter dot"
   },
   "files": [
-    "*.d.mts",
     "*.d.ts",
-    "*.js",
```

Comment:
was publishing a wrong duplicate file of "test-helpers.js", while actually we used the auto-generated mjs one